Changes.md

```diff
@@ -3,6 +3,11 @@
 
 > Caution : A bug fix in the handling of the `ai:volume:step_scale` attribute may change the appearance of Arnold volume renders.
 
+Features
+--------
+
+- VolumeScatter : Added a new node scattering points throughout a volume.
```

Comment:
a new node _for_ scattering points...

This is currently in a section for the upcoming 1.2.9 release, but the PR is targeted to `main` for the upcoming 1.3 release. It could go into either, but since there's no rush for it, I think perhaps it makes sense to put it in 1.3, paired with the improved Scatter?